### PR TITLE
Implement getting calendar event details

### PR DESCRIPTION
**Pull request by lunohodov**
This adds support for getting the details of a given calendar event from ESI.

Currently ESI has two endpoints which return event information
* `/characters/{character_id}/calendar` lists calendar event summaries
* `/characters/{character_id}/calendar/{event_id}` gets event details

We already use the `Event` model when listing calendar event summaries. Thus despite it's name, the model represents an event's summary and not an actual event.

A potential (re)use of the `Event` model in the context of getting event details may be confusing because:
* the endpoints respond with different data structures
* the endpoints use different names to convey the same information. For example `event_response` vs. `response`

Because of the above, we add a new `EventDetails` model to represent the details of an event.

### CODE REVIEW

File: lib/eve_online/esi/models/event_details.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          date = options['date']
+
+          parse_datetime_with_timezone(date) if date
+        end
+
+        def duration
+          duration = options['duration']
+
+          duration&.to_i
+        end
```
Comment (biow0lf) [inline lib/eve_online/esi/models/event_details.rb]: I think, duration should be just `options['duration']`

**Comment by biow0lf** [comment]
Anyway, awesome job!

**Comment by biow0lf** [comment]
And, add yourself to contributors in README.md

**Comment by lunohodov** [comment]
Do you want me to squash the fixups or you'll do it on merge?

**Comment by biow0lf** [comment]
@lunohodov I merge PR.

Also, I send you invite to this repo. If you are interesting in development of this gem, accept it. 👍 🔥

**Pull request merged**